Sources/Recoverable/Recoverable.swift

```diff
@@ -31,44 +29,74 @@ extension UIView: Recoverable {
         layer.cornerRadius = safeViewState.cornerRadius
         layer.masksToBounds = safeViewState.clipToBounds
         
+        startTransitionIfAvailable()
```

Comment:
I think this is not a right place to start the transition. I mean, here we are recovering the state, so we should move this code to the place where we hide the skeleton.